assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -121,9 +122,10 @@ function ExecutionResults({
       <ResultsContainer
         catalogError={false}
         clusterID={clusterID}
-        hasAlreadyChecksResults
+        hasAlreadyChecksResults={!!(executionData || executionLoading)}
         selectedChecks={checkResults}
         onCatalogRefresh={onCatalogRefresh}
+        usingNewChecksEngine="true"
```

Comment:
Change this by simply `usingNewChecksEngine`. That `"true"` is not even a real boolean hehe

Reply:
Yes, that's why you only need to put `usingNewChecksEngine` without anything as value:
```
<ResultsContainer
        catalogError={false}
        clusterID={clusterID}
        hasAlreadyChecksResults={!!(executionData || executionLoading)}
        usingNewChecksEngine
>
```

Reply:
Ah it's always true then... 
Thanks!